### DISCUSSION TITLE
Medium=Radio and missing LIT fields

### DISCUSSION THREAD

**Discussion by samsethi** [Podcastindex-org/podcast-namespace#679] [Ideas]
Good news we have [Fountain Radio working in TrueFans](https://truefans.fm/fountain-radio/675c06f8d55de476b7b25e56) with video  - see image 1 below.

<img width="1280" alt="Fullscreen_13_12_2024__10_53" src="https://github.com/user-attachments/assets/a1aefdca-c2bd-44f2-b6c4-47731aefaa0e" />

I think there needs to be an update to the LIT documentation.  Oscar has set the duration for 7 days in the RSS Feed.  Maybe we need to add a **continuous option for radio?**  - see image 2 below

<img width="1280" alt="LIT Opptions" src="https://github.com/user-attachments/assets/e84efef5-927b-4af7-80e4-2d24eaf167c5" />

Also should we add medium=radio in the feed, it is currently medium=podcast??? - see image 3 below.  Also for Frequency should we add an option for continuous when medium=radio? 

<img width="1429" alt="Fullscreen_14_12_2024__20_08" src="https://github.com/user-attachments/assets/2e8f0d5e-31aa-4f02-b0b0-dc011334a7ed" />

There is also no alternate enclosure in the feed for audio.  Is this a limitation of LIT?  Should we add Alternate Enclosure to LIT. 

We have also been working with [DRN1 Radio](https://truefans.fm/publishers/drn1) in Australia. They have added a [Publisher/Radio Feed](https://truefans.fm/publishers/drn1).  The feed also has LIT items for each show which means we are creating a real-time schedule. 

<img width="1320" alt="Cursor_and_TrueFans___Publishers___DRN1" src="https://github.com/user-attachments/assets/a83194f1-da08-4db1-a5e9-6c7b3377236a" />

How do we get these changes added? @daveajones

**Comment by Dwev** [comment]
Are these continuous radio feeds, enclosures in an RSS feed? If so, I like the idea of `medium=radio`, and "continuous" frequency. It makes a lot of sense for this medium that isn't like the others.

**Comment by samsethi** [comment]
I am starting a program in 2025 to get more live podcasts and radio programs on the platform using LIT. So medium=radio and continuous are two attributes we already support.

**Comment by agates** [comment]
For historical reference.

[Radio mediums #567](https://github.com/Podcastindex-org/podcast-namespace/discussions/567)
